cmd/image-builder/main.go

```diff
@@ -87,6 +90,37 @@ func main() {
 	}
 
 	echoServer := echo.New()
+	echoServer.HideBanner = true
+	echoServer.Logger = common.Logger()
+	echoServer.Use(middleware.RequestLoggerWithConfig(middleware.RequestLoggerConfig{
```

Comment:
So all other webserver config is currently located in v1/server.go in Attach() (things like errorhandler and Binder, stuff that's common between all echo groups).

I think it makes sense to extract all server-wide config (like logging, binder, errorhandler), since for v2 we'll need that in a common place. But let's do that in a followup.